openmetadata-ui/src/main/resources/ui/src/components/ManageTab/ManageTab.component.tsx

```diff
@@ -225,7 +226,17 @@ const ManageTab: FunctionComponent<Props> = ({
             theme="primary"
             variant="link"
             onClick={() => setListVisible((visible) => !visible)}>
-            {getOwnerById() || 'Select Owner'}
+            {getOwnerById() ? (
```

Comment:
We are calling `getOwnerById` thrice. Instead, we should call it once and store it in a variable and use that variable everywhere.